contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -328,30 +890,62 @@ avoid these error conditions are to:
 * Separate out volumes that the user prebinds from the volumes that are
 available for the system to choose from by StorageClass.
 
-#### PV Controller Changes
+### PV Controller Changes
 When the feature gate is enabled, the PV controller needs to skip binding
 unbound PVCs with VolumBindingWaitForFirstConsumer and no prebound PVs
 to let it come through the scheduler path.
 
 Dynamic provisioning will also be skipped if
```

Comment:
Dynamic provisioning will be skipped or delayed until `annSelectedNode` annotation is set? Does this mean dynamic provisioners should look for this annotation for the storage classes with `VolumeBindingMode` set to `VolumeBindingWaitForFirstConsumer` before they can act on the PVCs of that type?